controllers/grid/settings/category/form/CategoryForm.inc.php

```diff
@@ -292,6 +315,79 @@ function execute(...$functionArgs) {
 		parent::execute(...$functionArgs);
 		return $category;
 	}
-}
 
+	/**
+	 * Get a list of all subeditor IDs assigned to this category
+	 *
+	 * @param $categoryId int
+	 * @param $contextId int
+	 * @return array
+	 */
+	public function _getAssignedSubEditorIds($categoryId, $contextId) {
+		return Services::get('user')->getIds(array(
+			'contextId' => $contextId,
+			'roleIds' => ROLE_ID_SUB_EDITOR,
+			'assignedToCategory' => $categoryId,
+		));
+	}
+
+	/**
+	 * Compile data for a subeditors SelectListPanel
+	 *
+	 * @param $contextId int
+	 * @param $request Request
+	 * @return \PKP\components\listPanels\ListPanel
+	 */
+	public function _getSubEditorsListPanel($contextId, $request) {
+
+		$params = [
+			'contextId' => $contextId,
+			'roleIds' => ROLE_ID_SUB_EDITOR,
+		];
+
+		import('classes.core.Services');
+		$usersIterator = Services::get('user')->getMany($params);
+		$items = [];
+		foreach ($usersIterator as $user) {
+			$items[] = [
+				'id' => (int) $user->getId(),
+				'title' => $user->getFullName()
+			];
+		}
+
+		return new \PKP\components\listPanels\ListPanel(
+			'subeditors',
+			__('user.role.subEditors'),
```

Comment:
This should be a new locale key: "Category Editors".

Reply:
Actually, since the user group is still called Section Editors it could be confusing to call them Category Editors. Let's just call them "Assigned Editors" or "Editors". And apply that to the SectionForm as well.

Reply:
check